websites/H/Hoppscotch/lib/workspace.ts

```diff
@@ -0,0 +1,31 @@
+type Workspace =
+	| { kind: "Personal" }
+	| { kind: "Team"; name: string }
+	| { kind: "Error" };
+
+const getWorkspace = (): Workspace => {
+	// eslint-disable-next-line no-one-time-vars/no-one-time-vars
```

Comment:
Avoid disabling eslint rules unless it breaks how the code works. These rules are there to ensure code consistency across the project.